modules/system/assets/ui/docs/breadcrumb.md

```diff
@@ -4,10 +4,12 @@ Display a breadcrumb on the page.
 
     <div style="padding: 20px">
         <div class="control-breadcrumb">
-            <ul>
-                <li><a href="#">Dash Board</a></li>
-                <li><a href="#">Blog Posts</a></li>
-                <li>Edit Post</li>
-            </ul>
+            <nav aria-label="Breadcrumb">
```

Comment:
@ayumihamsaki why do you remove `control-breadcrumb` class?

Reply:
@w20k I didn't it on line 6 ?

Reply:
Sorry, my bad, morning sleepy eyes :)

Reply:
haha, I thought I completely missed something!